hack/make/tgz

```diff
@@ -9,6 +9,11 @@ if [ ! -d "$CROSS/linux/amd64" ]; then
 	false
 fi
 
+bundle .integration-daemon-start
+bundle .detect-daemon-osarch
+
+make manpages
```

Comment:
Isn't there a previous task that already generates these, or is that long-gone by now?  (ie, for binaries, this script simply includes binaries that have already been generated, so if they aren't generated, they don't get tarballs, and man pages _could_ be similar if we're already generating them elsewhere and thus would have them generated already during release and not have to build a new Docker image making the release process even longer than it already is :innocent:)